highfive/newpr.py

```diff
@@ -297,6 +300,11 @@ def get_irc_nick(gh_name):
             return rustacean_data[0].get("irc")
     return None
 
+# Find the contribution instructions link for the repo, if there is one
+def find_contrib_link(repo):
+    repoinfo = _load_json_file(repo + '.json')
```

Comment:
It would be better to only open the repo's json file once per request rather than multiple times (we already open it to get the reviewers list)